document/js-api/index.bs

```diff
@@ -1014,7 +1014,7 @@ Note: Exported Functions do not have a \[[Construct]] method and thus it is not
         1. If |method| is undefined, [=throw=] a {{TypeError}}.
         1. Let |values| be ? [=IterableToList=](|ret|, |method|).
         1. Let |wasmValues| be a new, empty [=list=].
-        1. Assert: |values| and |results| have the same length.
+        1. If |values| and |results| do not have the same length, throw a {{TypeError}}.
```

Comment:
Nit: `[=throw=]`.

Reply:
Weirdly, line 1014 is the only place in the spec we use the [=throw=] wording... IMO, it seems like we should refactor that in a follow up patch, if that's the preferred wording.

Reply:
Fine by me either way.